api/src/components/sitemap/controller.ts

```diff
@@ -0,0 +1,33 @@
+import * as I from 'interface';
+import * as response from 'lib/response';
+import * as sitemapService from 'sitemap/service';
+
+export const generate = async (): Promise<I.JSONResponse> => {
+    try {
+        const sitemapCount = await sitemapService.generatePublicationSitemaps();
+
+        return response.json(200, {
```

Comment:
This is not an API endpoint, it doesn't need to return anything

Reply:
Have tidied up the sitemap controller + service along these lines